lib/CapabilityDelegation.js

```diff
@@ -38,10 +38,14 @@ module.exports = class CapabilityDelegation extends ControllerProofPurpose {
    *   the creation of the proof.
    * @param [maxTimestampDelta] {integer} a maximum number of seconds that
    *   the date on the signature can deviate from, defaults to `Infinity`.
+   * @param {function} [inspectCapabilityChain] - An async function used to
+   *  check the capability chain. It can be use to find revocations related
+   *  to any of the capabilities in the chain. Tne expected return value
```

Comment:
Tne => The (same with two other doc updates).
(Input to function not documented here either, but can be addressed later).